### ISSUE TITLE
What do Prenet and Postnet used for?

### ISSUE THREAD

**Issue by youngsuenXMLY** [jjery2243542/adaptive_voice_conversion#13]
Hi, I see Prenet and Postnet code in model.py. Do Prenet used before both Ec and Es and Postnet used after Decoder?

**Comment by jjery2243542** [comment]
Hi,
It is just an experimental code. They didn't use anywhere.

**Comment by youngsuenXMLY** [comment]
An additional question, I didn't find any constraint on speaker encoder. I have an idea - a speaker encoder(Es) loss is added to keep speaker-embeddings invariant if Ec encodes Speaker1 and Es encodes Speaker2. The reconstructed X1_2 should have the same speaker-embeddings with Speaker2.  c1 = Ec(X1), s2 = Es(X2), X1_2 = D(c1,s2),
So the loss is torch.nn.L1loss(Es(X1_2)-s2)
Do you think this may help?
